### ISSUE TITLE
Contribution flow: step profile + payment accessibility issues

### ISSUE THREAD

**Issue by Betree** [opencollective/opencollective#3585]
On the step profile & step payment, using the tab button kind of works but doesn't show you which entry in the radio list is actually focused.

![image](https://user-images.githubusercontent.com/1556356/95590948-ffc4ad00-0a46-11eb-8561-0560e2226ee9.png)

**Comment by terminator0309** [comment]
@Betree can i work on this ...?
Looks not so difficult for me as i am first timer

**Comment by Betree** [comment]
Sure, some help on this would be much appreciated!

**Comment by terminator0309** [comment]
@Betree i am not able to change selected radio button using Tabs, I think i need to add onHover effect to those radio buttons.

**Comment by Betree** [comment]
Not `hover`, the event that we need to handle here is `focus`

**Comment by terminator0309** [comment]
If i select one of the radio buttons then i am able to navigate through them by using up/down arrow but tab button does nothing to them.

**Comment by terminator0309** [comment]
@Betree What exact behaviour do we looking for....?

**Comment by Betree** [comment]
We want to surface which element is currently focused. It's important for accessibility.

Basically, you should be able to complete these two steps by just using your keyboard, with the tab + arrows key.

So we need to come up with some CSS to reflect this. Below is a screencast of what with have with a similar radio list in another part of the website:
![Peek 2020-10-21 15-50](https://user-images.githubusercontent.com/1556356/96729228-38547700-13b5-11eb-9110-6d98c4721033.gif)

**Comment by terminator0309** [comment]
@Betree i dont know why but my radio button looks different. Am i missing some dependency..!?
![image](https://user-images.githubusercontent.com/57085155/97018662-0a199780-156d-11eb-859b-deeabd7cf44b.png)

**Comment by Betree** [comment]
As mentioned, the screencast above is from another part of the website.

Also, radio buttons styles can slightly vary across browsers.

**Comment by terminator0309** [comment]
@Betree i tried this but it has no effect on radio buttons. I am totally confused.
![image](https://user-images.githubusercontent.com/57085155/97262836-b26a7d00-1847-11eb-9c10-997b1edd65f8.png)

**Comment by Betree** [comment]
Sorry, I can't provide much more assistance. I recommend looking at the styles & behavior for the Github flow (where I took the screenshot from). You can access it from https://staging.opencollective.com/opensource/apply/ > Verify using github stars.

**Comment by Betree** [comment]
@terminator0309 I'm pushing an update for this as part of the recent contribution flow updates

**Comment by terminator0309** [comment]
@Betree I had tried that but it didn't seem to work. I am not very good with radio buttons. BTW thanks for your precious time.
